tap/settings.go

```diff
@@ -14,8 +14,8 @@ const (
 	MemoryProfilingTimeIntervalSeconds             = "MEMORY_PROFILING_TIME_INTERVAL"
 	MaxBufferedPagesTotalEnvVarName                = "MAX_BUFFERED_PAGES_TOTAL"
 	MaxBufferedPagesPerConnectionEnvVarName        = "MAX_BUFFERED_PAGES_PER_CONNECTION"
-	MaxBufferedPagesTotalDefaultValue              = 5000
-	MaxBufferedPagesPerConnectionDefaultValue      = 5000
```

Comment:
Should we set these values `0`? `0` is the default value inside the `gopacket`.

Reply:
Let's not change these consts with this PR. Reverted with https://github.com/up9inc/mizu/commit/3f6c98db13ba4d873562c9e3ba9f1c3239caa2a5